clients/drcachesim/tests/cache_replacement_policy_unit_test.cpp

```diff
@@ -140,14 +149,28 @@ unit_test_cache_lru_four_way()
     // Access the cache line in the following fashion. This sequence follows the
     // sequence shown in i#4881.
     // Lower-case letter shows the least recently used way.
-    cache_lru_test.access_and_check_cache(addr_vec[ADDR_A], 1); // A x X X
-    cache_lru_test.access_and_check_cache(addr_vec[ADDR_B], 2); // A B x X
-    cache_lru_test.access_and_check_cache(addr_vec[ADDR_C], 3); // A B C x
-    cache_lru_test.access_and_check_cache(addr_vec[ADDR_D], 0); // a B C D
-    cache_lru_test.access_and_check_cache(addr_vec[ADDR_A], 1); // A b C D
-    cache_lru_test.access_and_check_cache(addr_vec[ADDR_A], 1); // A b C D
-    cache_lru_test.access_and_check_cache(addr_vec[ADDR_A], 1); // A b C D
-    cache_lru_test.access_and_check_cache(addr_vec[ADDR_E], 2); // A E c D
+    cache_lru_test.access_and_check(addr_vec[ADDR_A], 1);     // A x X X
+    cache_lru_test.access_and_check(addr_vec[ADDR_B], 2);     // A B x X
+    cache_lru_test.access_and_check(addr_vec[ADDR_C], 3);     // A B C x
+    cache_lru_test.access_and_check(addr_vec[ADDR_D], 0);     // a B C D
+    cache_lru_test.access_and_check(addr_vec[ADDR_A], 1);     // A b C D
+    cache_lru_test.access_and_check(addr_vec[ADDR_A], 1);     // A b C D
+    cache_lru_test.access_and_check(addr_vec[ADDR_A], 1);     // A b C D
+    cache_lru_test.access_and_check(addr_vec[ADDR_E], 2);     // A E c D
```

Comment:
If you move the extra padding spaces inside the comment, e.g. from before to after the //, I think clang will allow them.

Reply:
Done. I don't think it's super pretty, but it's definitely readable. Thanks for the suggestion.